### PR TITLE
New license page

### PR DESCRIPTION
**Pull request by kerrywatson1**
@asmeurer please merge.

### CODE REVIEW

File: license.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
+==========================
+END USER LICENSE AGREEMENT
+==========================
+
+for Anaconda, Miniconda, and Conda
```
Comment (asmeurer) [inline license.rst]: The Anaconda license is different from the conda license. The conda license is the BSD license listed [here](https://github.com/conda/conda/blob/master/LICENSE.txt).

Comment (kerrywatson1) [inline license.rst]: Where does this appear in the program?

Comment (asmeurer) [inline license.rst]: I'm not sure what you mean by this.

Comment (kerrywatson1) [inline license.rst]: I need to redo that page anyway, thanks for pointing that out.

Kerry

**Pull request closed without merging**